### PR TITLE
Deploy update mode

### PR DESCRIPTION
**Pull request by pyseed**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ optional arguments:
   --master NODE         Node where to deploy the master (required for new
```
Comment (petrus-v) [inline README.md]: I guess ``[-u]`` is missing above

Comment (pyseed) [reply README.md]: hi! i do not see/match what is missing.? short and long option in readme. https://github.com/mlfmonde/cluster_cli/pull/12/commits/c30f7a95d0df25634aad822968aec1d6befda705

Comment (petrus-v) [reply README.md]: I couldn't comment but few lines above: https://github.com/mlfmonde/cluster_cli/blame/master/README.md#L99 should be updated, insn't it ?

Comment (pyseed) [reply README.md]: ok, got it !

Comment (pyseed) [reply README.md]: done

**Comment by petrus-v** [comment]
Thanks a lot
LGTM

**Pull request merged**